Documentation/operations/upgrade.rst

```diff
@@ -563,12 +575,60 @@ Once the migration is complete, confirm the endpoint identities match by listing
 Clearing CRD identities
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-If a migration has gone wrong, it possible to start with a clean slate. Ensure that no cilium instances are running with identity-allocation-mode crd and execute:
+If a migration has gone wrong, it possible to start with a clean slate. Ensure that no Cilium instances are running with ``--identity-allocation-mode=crd`` and execute:
 
 .. code-block:: shell-session
 
       $ kubectl delete ciliumid --all
 
+.. _double_write_migration:
+
+Migration with the "Double Write" identity allocation mode
```

Comment:
The docs here describe a KVStore -> CRD migration, but couldn't this technically also be used for a CRD -> KVStore migration?

Reply:
Yeah, it's possible to do this as well! I added an explicit note about it in the docs.